jscomp/bin/bsb.ml

```diff
@@ -9698,14 +9698,14 @@ let output_ninja
     let () =
       output_string oc ninja_required_version ;
       output_string oc "bs_package_flags = ";
-      output_string oc ("-bs-package-name "  ^ package_name);
+      output_string oc ("-bs-package-name "  ^ (Filename.quote package_name));
```

Comment:
Sorry the *package* I was referring to in this PR is the npm package name (in package.json).
It seems like `-bs-package-name` can accept white spaces in the name (or at least `npm run build` still works), that's why it is quoted.

Okay I'll attempt to solve this problem in the next commit.